### PR TITLE
Add progress bar to grid

### PR DESCRIPTION
**Pull request by jenniferboedker**
This PR adds the progress bar into the grid

### CODE REVIEW

**Comment by jenniferboedker** [comment]
[Mock View: Include progress bar in the grid](https://trello.com/c/n2PgkIye/795-mock-view-include-progress-bar-in-the-grid)

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy [hunk 1]
```diff
@@ -8,8 +8,11 @@ import com.vaadin.ui.Button
 import com.vaadin.ui.Grid
 import com.vaadin.ui.HorizontalLayout
 import com.vaadin.ui.Label
+import com.vaadin.ui.ProgressBar
 import com.vaadin.ui.VerticalLayout
+import com.vaadin.ui.renderers.ComponentRenderer
```
Comment (Steffengreiner) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:13]: Unused Import
```suggestion
```

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy [hunk 2]
```diff
@@ -8,8 +8,11 @@ import com.vaadin.ui.Button
...
 import com.vaadin.ui.renderers.DateRenderer
+import com.vaadin.ui.renderers.ProgressBarRenderer
 import com.vaadin.ui.renderers.TextRenderer
```
Comment (Steffengreiner) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:16]: Unused Import 
```suggestion
```

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy [hunk 3]
```diff
@@ -8,8 +8,11 @@ import com.vaadin.ui.Button
 import com.vaadin.ui.Grid
 import com.vaadin.ui.HorizontalLayout
 import com.vaadin.ui.Label
+import com.vaadin.ui.ProgressBar
```
Comment (Steffengreiner) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy:11]: Unused Import
```suggestion
```

**Review by Steffengreiner — APPROVED** [review]
LGTM! 🚀

**Pull request merged**